pkg/bindings/images/build.go

```diff
@@ -743,11 +743,36 @@ func nTar(excludes []string, sources ...string) (io.ReadCloser, error) {
 	return rc, nil
 }
 
+func errIfSymlinkOutsideRoot(root, path string) error {
+	pathInfo, err := os.Lstat(path)
+	if err == nil {
+		if pathInfo.Mode()&os.ModeSymlink == os.ModeSymlink {
+			// If resolved symlink is inside build context just return
+			resolvedPath, _ := filepath.EvalSymlinks(path)
```

Comment:
No handling for the error?

Reply:
Initially my thought was to ignore this error and fail in the two lines below, but this might contain meaningful error so handling it now.

Reply:
Ignoring it because of the reason here: https://github.com/containers/podman/pull/16315#issuecomment-1294506978